action.yml

```diff
@@ -1,9 +1,42 @@
-  name: 'Publishing Parasoft SOAtest results'
+  name: 'Run Parasoft SOAtest'
   description: 'A GitHub Action for running functional tests with Parasoft SOAtest and reviewing results directly in GitHub.'
   author: 'Parasoft'
   branding:
     icon: 'aperture'
     color: 'blue'
+  inputs:
+    installDir:
+      description: 'Installation folder of Parasoft SOAtest. If not specified, the soatestcli executable must be added to PATH.'
+      required: false
+    workingDir:
+      description: 'Working directory for running SOAtest.'
+      required: false
+      default: ${{ github.workspace }}
+    testConfig:
+      description: 'Test configuration to be used for test execution.'
+      required: false
+      default: 'user://Example Configuration'
+    resource:
+      description: 'The path to the test suite(s) relative to the workspace.'
```

Comment:
I think "Relative path(s) to the test suite(s) within the workspace." sounds better.